lucille-core/src/main/java/com/kmwllc/lucille/stage/AddRandomField.java

```diff
@@ -146,7 +146,7 @@ private List<String> getUniqueValues(boolean dataExists, List<String> inputData)
       }
     } else {
       // create sequential list of numbers ending at range size
-      List<Integer> seqList = IntStream.rangeClosed(1, rangeSize)
+      List<Integer> seqList = IntStream.range(1, rangeSize)
```

Comment:
we still want to make sure the list's size is equal to rangeSize
so should the lower value be 0?

Reply:
Yea, I thought this was for IDs, but this is just the data that goes inside the random field so makes sense for it to be 0